code/datums/elements/voucher_redeemer.dm

```diff
@@ -62,14 +62,15 @@
 
 	var/selection = show_radial_menu(redeemer, source, cached_options, custom_check = CALLBACK(src, PROC_REF(check_menu), voucher, redeemer), radius = 38, require_near = TRUE, tooltips = TRUE)
 	if(!selection)
-		return
+		return ITEM_INTERACT_FAILURE
```

Comment:
This is an async proc the return value doesn't get sent anywhere ❓

Reply:
![i50pht3vdi6c1](https://github.com/user-attachments/assets/1d593901-775f-4d88-bf07-e07c76a1d2af)